src/modm/ui/color/gray.hpp

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2021, Thomas Sommer
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+#include <algorithm>
+#include <concepts>
+#include <functional>
+
+#include <modm/math/saturation/saturated.hpp>
+#include <modm/math/proportional_unsigned.hpp>
+
+#include <modm/math/utils/arithmetic_traits.hpp>
+#include <modm/math/utils/misc.hpp>
+
+#include "concepts.hpp"
+
+#include <modm/debug/logger.hpp>
+
+namespace modm::color {
+/**
+ * @brief 		Unsigned integer with arbitrary number of digits, proportional conversion
+ * 				and saturating arithemtics.
+ *
+ * @tparam D 	Number of Digits
+ * @tparam P	When forming a Buffer with this color, this would be the internal storage.
+ * 				Should be MCUs fastest unsigned int -> just keep the default.
+ * 				Some Displays require a specific type therefore it's a template argument.
+ *
+ *
+ * @author		Thomas Sommer
+ * @ingroup		modm_color_gray
```

Comment:
```suggestion
 * @ingroup		modm_ui_color
```
Unless I missed a `@defgroup modm_color_gray` somewhere (which is not recommended).

Reply:
This wasn't intended. thx